core/src/test/scala/org/apache/spark/CheckpointSuite.scala

```diff
@@ -618,4 +618,26 @@ class CheckpointCompressionSuite extends SparkFunSuite with LocalSparkContext {
       assert(rdd.collect().toSeq === (1 to 20))
     }
   }
+
+  test("cache checkpoint preferred location") {
+    withTempDir { checkpointDir =>
+      val conf = new SparkConf()
+        .set("spark.rdd.checkpoint.cachePreferredLocs", "true")
```

Comment:
Shall we use `CACHE_CHECKPOINT_PREFERRED_LOCS.key`? Otherwise, someone will create a follow-up later.

Reply:
Ok. Let me change it.